packages/ui-navigation/src/AppNav/Item/index.tsx

```diff
@@ -62,8 +63,7 @@ class Item extends Component<AppNavItemProps> {
 
   static defaultProps = {
     children: null,
-    // @ts-expect-error ts-migrate(6133) FIXME: 'event' is declared but its value is never read.
-    onClick: function (event) {},
+    onClick: function () {},
```

Comment:
You can remove onClick from the defaultProps, there is no point of it having a default anymore.

(It doesn't make any difference in the code, and the reason why we added them to defaultProps before was to be able to show the function signature -> those are rendered from the TS types now so no need to keep them in defaultProps)